lang/en/docs/yarnrc.md

```diff
@@ -56,7 +56,7 @@ Value must be a boolean, defaults to `false`.
 child-concurrency #number#
 ```
 
-Controls the number of child processes run parallely to build node modules.
+Controls the number of child processes run parallelly to build node modules.
```

Comment:
```suggestion
Controls the number of child processes run in parallel to build node modules.
```

?

Reply:
Controls the number of child processes **that** run in parallel to build node modules.

WDYT @Haroenv?

Reply:
yes, that seems best, thanks!